server/DataAccessObjects/GroceryStoreDao.js

```diff
@@ -47,17 +47,24 @@ class GroceryStoreDao {
         var updateItems = {};
         for (const [itemId, item] of Object.entries(orderInventory)) {
             var remainingQuantity = Number(groceryStoreInventory[itemId]["quantity"]) - item.getQuantity();
-            updateItems[item.getId()] = {
-                "id": item.getId(),
-                "name": item.getName(),
-                "brand": item.getBrand(),
-                "groceryStoreId": item.getGroceryStoreId(),
-                "quantity": remainingQuantity,
-                "expirationDate": item.getExpirationDate(),
-                "ediOrderNumber": item.getEdiOrderNumber()
-            };
+            if (remainingQuantity == 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/gh/CSC492-DSC-Community-Projects/Backend/pullRequest?prid=5262914)